lms/static/sass/application.scss.mako

```diff
@@ -43,6 +43,7 @@
 @import 'elements/controls';
 
 // shared - course
+@import 'shared/fields';
 @import 'shared/forms';
```

Comment:
I missed this in my initial review, but ideally we would actually be building on the work done in _forms.scss for these fields. For now I'm adding a story to the UX backlog for this cleanup instead of adding it here, in part because _forms.scss doesn't have a lot of what is necessary for this PR anyways. https://openedx.atlassian.net/browse/UX-1856